notebook/analysis/benchmark_results.ipynb

```diff
@@ -10517,12 +10517,1360 @@
     "plot"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sex Prediction"
+   ]
+  },
```

Comment:
Do you want something in this cell? I just see an empty cell after the "Sex Prediction" heading.

Reply:
Nope, added a commit to remove the empty cell